### PR TITLE
Project setup

### PR DESCRIPTION
**Pull request by henokkhm**
This pull request initializes the Bookstore project with Create React App. It sets up client-side routing using React Router version 6. The app currently has two routes, which both render a page component:
- `/` the homepage
- `/categories` the categories page

**Summary**

- Create README.md using the Microverse template
- Add ESLint, Stylelint, and GitHub Action workflow for linters
- Add react-router-dom 
- Add home and categories pages
- Add CSS reset
- Add Nunito Sans from Google Fonts
- Add Navbar component
- Add global CSS
- Add placeholder books' data
- Add BookList component
- Add Book component
- Add the 'Add new book form' component
- Add general styles to all components

### CODE REVIEW

File: src/components/AddBookForm.js [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+            className={styles.inputs}
+            type="number"
+            id="percentCompleted"
+            name="percentCompleted"
+            placeholder="50"
+            min="0"
+            max="100"
+            required
+          />
+        </label>
```
Comment (Tufahel) [inline src/components/AddBookForm.js]: - [x] As per the project requirements `Category` should be replaced by `Author`, and in the design template there is no input name as `Precent completed`. So, kindly remove those two input to meet the requirements.

image

File: README.md [hunk 2]
```diff
@@ -1 +1,181 @@
...
+
+I would like to thank Microverse for teaching and enforcing professional level practices.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LICENSE -->
+
+## 📝 License <a name="license"></a>
+
+This project is [MIT](./LICENSE) licensed.
```
Comment (Tufahel) [inline README.md:179]: - [x] You haven't added the `LICENCE.md` file that's why it is showing errors. Kindly fix this issue to make your readme more professional.

image

**Review by Tufahel — CHANGES_REQUESTED** [review]
# Status: Required Changes :recycle:
Hi @henokkhm :wave:  

Good job so far! 👍 
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights 🌟

Alt Text

✔️Contains individual book state, as well as a button to delete the book
✔️Contains a list to renders individual books
✔️Contains a form to create new books
✔️Contains a navigation to link to the 2 routes you created

## Required Changes ♻️

***Please kindly check the comments/review below.***

## Optional suggestions 🤷‍♂️

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

## Important Notes 📝
- Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
- Please, remember to tag me in your question so I can receive the notification.
- You can mail me at [Mail](mailto:tufahel96@gmail.com) or ask any question at Slack, `Name: Tufahel`

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Felix45 — APPROVED** [review]
# Approved :green_circle: 
Hi @henokkhm, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights :heavy_check_mark: 
- [x] Bookstore components added as expected :white_check_mark: 
- [x] Application uses react-router for navigation :white_check_mark: 
- [x] Pull request has a descriptive title and summary :white_check_mark: 
- [x] Git flow is used correctly :white_check_mark:   

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**